### PR TITLE
proto command for testing purposes

### PR DESCRIPTION
**Pull request by yisraelU**
adds command to handle cli input as smithy string and attempts to generate scala code .
2 arguments , input and output directory(defaults to pwd)

p.s. 
i don't like the term `proto`,perhaps `test` would be better, or maybe you have a better term

### CODE REVIEW

**Comment by Baccata** [comment]
Yeah `proto` is not a great name in that it would conflate with the proto IDL.

File: modules/codegen-cli/src/smithy4s/codegen/cli/ProtoTypeCommand.scala [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import os.Path
+import smithy4s.codegen.ProtoTypeArgs
+import smithy4s.codegen.cli.Smithy4sCommand.ProtoType
+
+object ProtoTypeCommand {
+
+  import Options._
+
+  val smithyString =
+    Opts.option[String](long = "input", help = "Smithy string ")
```
Comment (Baccata) [inline modules/codegen-cli/src/smithy4s/codegen/cli/ProtoTypeCommand.scala:32]: Don't think this should be an `Opts.option` but rather an `Opts.argument`

**Comment by Baccata** [comment]
I'm cool with the addition of a command, just needs a better name :)

**Comment by Baccata** [comment]
Any thoughts of whether to update or close this ?

**Comment by yisraelU** [comment]
I am going to close this for now, I think I might like something like a repl mode in the future

File: modules/codegen/src/smithy4s/codegen/CodegenArgs.scala [hunk 2]
```diff
@@ -28,3 +28,8 @@ case class CodegenArgs(
     dependencies: List[String],
     transformers: List[String]
 )
+
+case class ProtoTypeArgs(
+                          smithyString:String,
```
Comment (kubukoz) [inline modules/codegen/src/smithy4s/codegen/CodegenArgs.scala:33]: Could we achieve the same outcome by using the `codegen` command with `-` as the input path, and piping the string through stdin?

**Pull request closed without merging**